### PR TITLE
[Docs] Update compressed_tensors.md

### PR DESCRIPTION
**Pull request by mgoin**
Fix some unfinished sections

# What does this PR do?

Fixes issues in the https://huggingface.co/docs/transformers/en/quantization/compressed_tensors docs

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by ArthurZucker** [comment]
Ping us when ready! 🤗

File: docs/source/en/quantization/compressed_tensors.md [hunk 1]
```diff
@@ -19,23 +19,20 @@ The [`compressed-tensors`](https://github.com/neuralmagic/compressed-tensors) li
...
 1. `dense`
-2. `int-quantized`: INT8 quantized models
-    - sample [model/config](https://huggingface.co/nm-testing/tinyllama-w8a8-compressed-hf-quantizer)
-3. `float-quantized`: FP8 quantized models; currently support E4M3
-    - sample [model/config](https://huggingface.co/nm-testing/Meta-Llama-3-8B-Instruct-fp8-hf_compat/tree/main)
-4. `pack-quantized`: INT4 or INT8 weight-quantized models, packed into INT32. For INT4, the weights have an INT4 range but are stored as INT8 and   then packed into INT32.
-    - sample [model/config](nm-testing/tinyllama-w4a16-compressed-hf-quantizer)
+2. `int-quantized` ([sample](https://huggingface.co/nm-testing/tinyllama-w8a8-compressed-hf-quantizer)): INT8 quantized models
+3. `float-quantized` ([sample](https://huggingface.co/nm-testing/Meta-Llama-3-8B-Instruct-fp8-hf_compat)): FP8 quantized models; currently support E4M3
+4. `pack-quantized` ([sample](nm-testing/tinyllama-w4a16-compressed-hf-quantizer)): INT4 or INT8 weight-quantized models, packed into INT32. For INT4, the weights have an INT4 range but are stored as INT8 and then packed into INT32.
```
Comment (yuanx749) [inline docs/source/en/quantization/compressed_tensors.md:24]:
```suggestion
4. `pack-quantized` ([sample](https://huggingface.co/nm-testing/tinyllama-w4a16-compressed-hf-quantizer)): INT4 or INT8 weight-quantized models, packed into INT32. For INT4, the weights have an INT4 range but are stored as INT8 and then packed into INT32.
```
The original link redirects to a 404 page.

**Review by yuanx749 — CHANGES_REQUESTED** [review]

File: docs/source/en/quantization/compressed_tensors.md (hunk 1 above)
Comment (mgoin) [reply docs/source/en/quantization/compressed_tensors.md:24]: Yep, this is why I updated it to a valid link

**Review by kylesayrs — APPROVED** [review]

**Comment by mgoin** [comment]
Hi @ArthurZucker we are ready to go, thanks!

**Review by SunMarc — APPROVED** [review]
LGTM! Thanks!

**Review by ArthurZucker — APPROVED** [review]
🤗

**Pull request merged**